managed/src/main/java/com/yugabyte/yw/controllers/CertificateController.java

```diff
@@ -129,11 +131,18 @@ public Result getRootCert(UUID customerUUID, UUID rootCA) {
   }
 
   public Result list(UUID customerUUID) {
-    List<CertificateInfo> certs = CertificateInfo.getAll(customerUUID);
-    if (certs == null) {
+    List<CertificateInfo> certificates = CertificateInfo.getAll(customerUUID);
+    ArrayNode certList = Json.newArray();
+    certificates.forEach((certificate) -> {
```

Comment:
You don't need to do this. Since we added a getter method in the CertificateInfo class, it will automatically populate that in the object's json.

Reply:
I can not see this field in the response without this. Am I missing anything here? any annotations required for the method? @Arnav15

Reply:
@mahendranbhat two things here.
1) For the getInUse method, the customerUUID should not be passed in as a parameter. Since CertificateInfo already has customerUUID as a class field, just use that in the function instead of having a customerUUID parameter. 

2) Once you do this and the getter does not have a parameter, it should automatically add inUse to the json object without you needing to do it here as @Arnav15 mentioned.